esm/interface/css-style-declaration.js

```diff
@@ -81,6 +81,21 @@ export class CSSStyleDeclaration extends Map {
     refs.get(this).setAttribute('style', value);
   }
 
+  getPropertyValue(name) {
+    const self = this[PRIVATE];
+    return handler.get(self, name);
+  }
+
+  setProperty(name, value) {
+    const self = this[PRIVATE];
+    handler.set(self, name, value);
```

Comment:
this is fine only because there's quite some logic in the handler ... when it comes to the getter there's nothing else than a *map.get*